### PR TITLE
[DOCS-5459] Application upgrades

### PR DESCRIPTION
**Pull request by m-hulbert**

### CODE REVIEW

File: docs/administrator/deploy/upgrade.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+# Upgrading applications
+Upgrading an application allows for a new version of a released project to be deployed to an existing application. Tasks and process instances that are in progress and based on a previous application version can still be completed, however any new ones started will use the new model definitions. 
+
+Some considerations for upgrading applications to be aware of are that:
+
+* An application can be upgraded to a released project version that is lower than the one currently deployed, however the application version will still increment.   
+* Applications can only be upgraded through the deployment service. This means that if an application was deployed via Helm it cannot be upgraded.
+* [UIs](../../modeling/interfaces.md) will be added and removed between versions based on whether they exist in the project. If only the `type` is changed and the `name` remains the same then the `type` will not be updated between versions. 
+* It is not possible to change an application name when upgrading an application.
```
Comment (ffazzini) [inline docs/administrator/deploy/upgrade.md]: i am not sure we should add this line, they ui won't allow you to change the app name anyway

File: docs/administrator/deploy/upgrade.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+| 5 | 4 | 
+| 3 | 5 |
+| 10 | 6 | 
+
+## Upgrade steps 
+Use the following steps to upgrade an application in the Administrator Application: 
+
+1. Sign into the Administrator Application. 
+2. Select the **Upgrade** action against an application in the **Application Instances** tab to upgrade it. 
+3. The application name can't be changed and the released project version to upgrade to must be greater than the one currently deployed.
```
Comment (ffazzini) [inline docs/administrator/deploy/upgrade.md]: no this point is not valid anymore, above you have already explained correctly

File: docs/administrator/deploy/upgrade.md (hunk 1 above)
Comment (m-hulbert) [reply docs/administrator/deploy/upgrade.md]: Good point - I've removed it.

File: docs/administrator/deploy/upgrade.md (hunk 2 above)
Comment (m-hulbert) [reply docs/administrator/deploy/upgrade.md]: Ah, yes that's true. I've removed this line too.

**Review by ffazzini — APPROVED** [review]

**Pull request merged**